configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -1774,6 +1774,10 @@ content.mixedContent.forceLinksInNewWindow=true
 # calendar.external.subscriptions.user.cacheentries=32
 # calendar.external.subscriptions.user.cachetime=120
 
+#Determine the calendar export period
```

Comment:
You should have the default value documented in here and a line ready to uncomment.